.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -297,3 +363,38 @@ sqs_Scenario_SendReceiveBatch:
                 - python.example_code.sqs.Scenario_SendReceiveBatch
   services:
     sqs: {}
+sqs_AuthorizeS3ToSendMessage:
+  title: Authorize an &S3; bucket to send messages to an &SQS; queue.
+  title_abbrev: Authorize a bucket to send messages to a queue
+  synopsis: authorize an &S3; bucket to send messages to an &SQS; queue.
+  category:
+  languages:
+      .NET:
+        versions:
+        - sdk_version: 3
+          github: dotnetv3/SQS
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+              - SQS.dotnetv3.AuthorizeS3ToSendMessageExample
+  services:
+    sqs: {AuthorizeS3ToSendMessage}

```

Comment:
This is an odd one--this appears to be a method that's special to .NET and not part of the service API, so while it does exist in the .NET reference, the goto link doesn't resolve to anything. For now, I think the answer is to remove the action, so:
```
sqs: {}
```